examples/basics.rs

```diff
@@ -15,9 +15,10 @@ fn grad_example() {
 }
 
 fn main() {
-    println!("Cuda available: {}", tch::Cuda::is_available());
-    println!("Cudnn available: {}", tch::Cuda::cudnn_is_available());
-    let device = tch::Device::cuda_if_available();
+    //println!("Cuda available: {}", tch::Cuda::is_available());
+    //println!("Cudnn available: {}", tch::Cuda::cudnn_is_available());
```

Comment:
If these two lines are uncommented, I get the following compile error:
```
error: linking with `cc` failed: exit status: 1
  = note: Undefined symbols for architecture arm64:
            "torch::cuda::is_available()", referenced from:
                _atc_cuda_is_available in libtorch_sys-0d280fb35559980b.rlib(torch_api.o)
            "torch::cuda::cudnn_is_available()", referenced from:
                _atc_cudnn_is_available in libtorch_sys-0d280fb35559980b.rlib(torch_api.o)
          ld: symbol(s) not found for architecture arm64
          clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

I'm not sure why. It doesn't happen on the macOS build.